js/player.js

```diff
@@ -0,0 +1,9 @@
+//A player object to store game info
+let player = {
```

Comment:
let me know if I'm wrong, but we should be able to keep this as a const since once we create player, it will never need to be changed.

Reply:
The stats on the player change though. So in order to mutate player's data, let needs to be used.

Reply:
it's semantics, but you can modify properties of an object just fine, it's only if you try to make player equal a new object that it errors out.

Reply:
OK. Didn't know about being able to change properties. I'll make that change.